js/app/restaurant/OrderList.jsx

```diff
@@ -1,55 +1,99 @@
 import React from 'react';
-import OrderListItem from './OrderListItem.jsx';
+import OrderLabel from '../order/Label.jsx';
 import _ from 'lodash';
 import moment from 'moment';
 
 class OrderList extends React.Component
 {
   constructor(props) {
-    super(props);
+    super(props)
     this.state = {
       orders: props.orders,
-    };
+      active: props.active
+    }
+  }
+
+  setActive(order) {
+    this.setState({ active: order })
   }
+
   addOrder(order) {
-    let { orders } = this.state;
-    orders.push(order);
 
-    this.setState({ orders });
+    let { orders } = this.state
+    orders.push(order)
+
+    this.setState({ orders })
+  }
+
+  renderOrders(date, orders) {
+    return (
+      <div key={ date }>
+        <h4>{ _.startCase(moment(date).calendar().split(' ')[0]) }</h4>
+        { this.renderOrdersTable(orders) }
+      </div>
+    )
+  }
+
+  renderOrdersTable(orders) {
+    return (
+      <div>
+        <table className="table table-hover">
+          <tbody>
+          { _.map(orders, (order) => this.renderOrderRow(order)) }
+          </tbody>
+        </table>
+      </div>
+    )
   }
+
+  renderOrderRow(order) {
+
+    let className = ''
+    if (this.state.active && this.state.active['@id'] === order['@id']) {
+      className = 'active'
+    }
+
+    return (
+      <tr key={ order['@id'] } onClick={ () => this.props.onOrderClick(order) } style={{ cursor: 'pointer' }} className={ className }>
+        <td>#{ order['@id'].replace('/api/orders/', '') }</td>
+        <td><OrderLabel order={ order } /></td>
+        <td><i className="fa fa-clock-o" aria-hidden="true"></i>  { moment(order.delivery.date).format('lll') }</td>
```

Comment:
IMHO we shouldn't show the delivery date but the date the dish has to be prepared (i.e. when the courrier will come).

For now on we have a fixed preparation time of 30 min and delivery time of 15 min [here](https://github.com/coopcycle/coopcycle-web/wiki/Business-assumptions) so you should show `deliveryDate - 30 min.`